content/en/apps/tutorials/death-reporting.md

```diff
@@ -1,38 +1,44 @@
 ﻿# Building A Death Report Workflow
+
 #### Guide for setting up a comprehensive death report workflow.
 
 This tutorial will take you through setting up a comprehensive death report workflow. This includes laying out a death report form as well as handling all the configurations needed for wiring it up in the CHT.
 By the end of the tutorial you should be able to:
--   View the death report menu
--   Open, fill and submit a death report form for a contact
--   Display all the deceased contacts of an area
+
+- View the death report menu
+- Open, fill and submit a death report form for a contact
+- Display all the deceased contacts of an area
 
 ## Required resources
 
 Before you begin you need to :
 
-1.  have a working CHT development environment (see docs or this)
-2.  Know how to build an app form in the CHT ([https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/))
-3.  know how to set form properties ([https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/](https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/))
-4.  know how to configure tasks ([https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/](https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/))
-5.  know about targets ([https://docs.communityhealthtoolkit.org/apps/tutorials/targets/](https://docs.communityhealthtoolkit.org/apps/tutorials/targets/))
+1. have a working CHT development environment (see docs or this)
+2. Know how to build an app form in the CHT ([https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/))
+3. know how to set form properties ([https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/](https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/))
+4. know how to configure tasks ([https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/](https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/))
+5. know about targets ([https://docs.communityhealthtoolkit.org/apps/tutorials/targets/](https://docs.communityhealthtoolkit.org/apps/tutorials/targets/))
 
 We will briefly explain most of these concepts on the fly, but it is important to understand them well in order to get the most out of this guide.
 
 ## Implementation Steps
+
 ### 1. Layout the form
+
 To create the form view, the first step is to create the XLSX file that best matches the proposed design. It should be noted that a death report form is not very long (usually two pages, one for filling in the elements of the report and another for the summary). An example death form XLSX file below:
```

Comment:
Use [active voice](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#active-voice) whenever you can. I'll also be reviewing the rest of the tutorials to follow the style guide.